src/main/java/MinesweeperController.java

```diff
@@ -228,4 +255,9 @@ private void gameOver() {
             }
         });
     }
+
+    private void setImageSize(ImageView image){
```

Comment:
I don't know why you're doing this in `revealCell()`

Reply:
I tried doing it in the CSS but it didn't work